### PR TITLE
[Locust] fix locust's dockerfile and improve generate_certs to associate a device with a certificate

### PR DESCRIPTION
**Pull request by ctossato**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
fix locust's dockerfile and improve generate_certs to associate a device with a certificate

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Yes. Locust will be compatible only with version 0.7.0 or higher.

* **Other information**:
Dockerfile was updated with following tag: 
https://github.com/docker-library/python/blob/fc33184453f89414b13a87904237fac90778ebeb/3.6/alpine3.14/Dockerfile

### CODE REVIEW

**Review by joelpmonteiro — COMMENTED** [review]
bom trabalho, eu também identifiquei que esta url foi preterida.

**Review by joelpmonteiro — DISMISSED** [review]
good job, fixing the locust building, and updating the Alpine version to use another version of python

File: connector/mqtt/locust/src/dojot/api.py [hunk 1]
```diff
@@ -252,6 +252,26 @@ def generate_certificate(jwt: str, csr: str) -> str:
 
         return (res['certificateFingerprint'], res['certificatePem'])
 
+    @staticmethod
+    def associate_device_with_certificate(jwt: str, device_id, fingerprint):
```
Comment (rascaraficci) [inline connector/mqtt/locust/src/dojot/api.py:256]: Não seria: `def associate_device_with_certificate(jwt: str, device_id: str, fingerprint: str)`?

File: connector/mqtt/locust/src/ejbca/certificate.py [hunk 2]
```diff
@@ -61,7 +61,11 @@ def generate_certificate(self) -> str:
 
         Returns the pem certificate.
         """
-        return DojotAPI.generate_certificate(self.jwt, self.csr["pem"])
+
+        fingerprint, pem_crt = DojotAPI.generate_certificate(self.jwt, self.csr["pem"])
+        DojotAPI.associate_device_with_certificate(self.jwt, self.c_name, fingerprint)
```
Comment (rascaraficci) [inline connector/mqtt/locust/src/ejbca/certificate.py:66]: Uma dúvida, o c_name corresponde ao deviceID ou tenant:deviceID?

**Review by rascaraficci — COMMENTED** [review]
I left minor comments.

File: connector/mqtt/locust/src/dojot/api.py (hunk 1 above)
Comment (ctossato) [reply connector/mqtt/locust/src/dojot/api.py:256]: sim. vou corrigir isso.

File: connector/mqtt/locust/src/ejbca/certificate.py (hunk 2 above)
Comment (ctossato) [reply connector/mqtt/locust/src/ejbca/certificate.py:66]: corresponde ao deviceID. Vou deixar esse ponto do código mais claro, que ai é o deviceID.

**Review by rascaraficci — APPROVED** [review]
Good job! It looks great to me!

**Comment by joelpmonteiro** [comment]
good work

**Review by joelpmonteiro — APPROVED** [review]
good work.
no comment

**Comment by ctossato** [comment]
this PR fix the issue dojot/dojot#2189

**Pull request merged**